pkgs/applications/window-managers/sway/swayest-workstyle.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "swayest-workstyle";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Lyr-7D1h";
+    repo = "swayest_workstyle";
+    rev = version;
+    sha256 = "sha256-LciTrYbmJV0y0H6QH88vTBXbDdDSx6FQtO4J/CFLjtY=";
+  };
+
+  cargoSha256 = "sha256-34Ij3Hd1JI6d1vhv1XomFc9SFoB/6pbS39upLk+NeQM=";
+
+  doCheck = false; # No tests
+
+  meta = with lib; {
+    description = "Map sway workspace names to icons defined depending on the windows inside of the workspace";
+    homepage = "https://github.com/Lyr-7D1h/swayest_workstyle";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ miangraham ];
```

Comment:
Add yourself to [`maintainer-list.nix`](https://github.com/NixOS/nixpkgs/blob/5f43d8b088d3771274bcfb69d3c7435b1121ac88/maintainers/maintainer-list.nix) if you aren't already (you aren't in it on my local checkout)

Reply:
I'm in there on master! :) https://github.com/NixOS/nixpkgs/blob/5df5df389a4a7a4395c5dc0d9359a85bbafd2c89/maintainers/maintainer-list.nix#L8546

Reply:
neat, accidentally checked out a 2 month old `nixos-unstable`